commands/Minigame.js

```diff
@@ -180,20 +177,23 @@ export default class Minigame extends Command {
   }
 
   clean() {
-    yuuto.off("messageReactionAdd", this.reactionRecvBound);
-    yuuto.off("message", this.messageRecvBound);
+    yuuto.off("messageReactionAdd", this.reactionRecv, this);
+    yuuto.off("message", this.messageRecv, this);
 
     this.state = State.OFF;
   }
 
   getScoreboard() {
-    const sortedPlayers = Array.from(this.players.keys()).sort((a, b) => {
-      return this.players.get(b) - this.players.get(a);
-    });
+    // Map#entries returns an array of [key, value]
+    const sortedPlayers = Array.from(this.players.entries()).sort(
+      ([, a], [, b]) => {
+        return b - a;
+      }
```

Comment:
This just looks better in this case imo
```suggestion
      (a, b) => {
        return b[1] - a[1];
      }
```